README.md

```diff
@@ -3,77 +3,75 @@
 <img width="105" height="35" alt="JWT Auth for open source projects" src="https://cdn.auth0.com/oss/badges/a0-badge-dark.png"></a>
 
 # Readme
+
 This is the repo for the new mobile web app of [Boxtribute](www.boxtribute.org), consisting of a [React front-end](/react) bootstrapped with [Create React App](https://github.com/facebook/create-react-app), and a [Python flask back-end](/flask).
 
 Please check out [**Contribution Guidelines**](CONTRIBUTING.md) before you get started!
 
+## Table of Contents
+
+1. [Contribution Guidelines](CONTRIBUTING.md)
+1. [Installation](#preparation-for-installation)
+   1. [Basic steps](#preparation-for-installation)
+   2. [Frontend](/react/README.md)
+   3. [Backend](/flask/README.md)
+1. [About Docker](#about-docker)
+1. [Development Database Seed](#development-database-seed)
+1. [CircleCI](#circleci)
+1. [Architecture overview](#architecture-overview)
+
 ## Preparation for Installation
 
-* Install [Docker](https://www.docker.com/products/docker-desktop)
-* Get in touch with the [Boxwise team](mailto:hello@boxwise.co) to get access to the [Auth0](https://auth0.com/) development tenant.
+- Install [Docker](https://www.docker.com/products/docker-desktop)
+- Get in touch with the [Boxwise team](mailto:hello@boxwise.co) to get access to the [Auth0](https://auth0.com/) development tenant.
 
 ## How do I get set up?
 
-1. Create a .env-file with the environment variables. Therefore, copy/paste `example.env` and rename it `.env`
+These are the most basic steps to get frontend and backend up and running. At the end, there are also further instructions to set-up your frontend and backend environment.
```

Comment:
Wait, isn't the first and second statement the same? Are you saying the further instructions will configure more complex features for the environment?

Reply:
updated. clarified